crates/ruff/src/rules/flake8_pyi/rules/simple_defaults.rs

```diff
@@ -523,3 +599,40 @@ pub(crate) fn unassigned_special_variable_in_stub(
         stmt.range(),
     ));
 }
+
+/// PIY026
+pub(crate) fn type_alias_without_annotation(checker: &mut Checker, value: &Expr, targets: &[Expr]) {
```

Comment:
I renamed the method and rule to be a little more in-line with our conventions ("describe the bad case").